core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/docker/DockerContainer.scala

```diff
@@ -257,17 +257,43 @@ class DockerContainer(protected val id: ContainerId,
    * previous activations that have to be skipped. For this reason, a starting position
    * is kept and updated upon each invocation.
    *
-   * If asked, check for sentinel markers - but exclude the identified markers from
-   * the result returned from this method.
+   * There are two possible modes controlled by parameter waitForSentinel:
    *
-   * Only parses and returns as much logs as fit in the passed log limit.
+   * 1. Wait for sentinel: tail container log file until two sentinel markers show up. Complete
```

Comment:
Please break the description to the next line. Indent by one or two spaces if you feel the necessity there.

Reply:
Makes sense. Re-formatting as suggested.